src/fauxtoshop.cpp

```diff
@@ -342,16 +304,19 @@ void getRandomPixelColor(Grid<int> &original
  * Returns: nothing. Void function
 */
 
-void applyScatterFilter(Grid<int> &original){
+void applyScatterFilter(GBufferedImage &img){

```

Comment:
Now that `original` is defined in the scope of this function, it's no longer needed as a parameter. Instead the `GBufferedImage` is passed in as a parameter since `original` is constructed from that.